src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -121,6 +133,10 @@ public function handleException($exception, $request, $response)
      */
     protected function getRenderer($exception)
     {
+        if (!$this->renderer) {
+            $this->renderer = $this->config('exceptionRender') ?: ExceptionRenderer::class;
+        }
```

Comment:
No, I only wanted to not have it outside to be a requirement to be passed in :) See the linked app PR